groovy/libs.groovy/external/groovy-2.5.2-license.txt

```diff
@@ -208,135 +209,13 @@ Origin: http://groovy-lang.org/
 
 ------------------------------------------------------------------------
 
-Asciidoc License
-
-This product uses the style.css from asciidoctor.org within
-documentation. The file is available under the MIT License.
-For details, see licenses/asciidoc-style-license.txt.
-
-The MIT License
-
-Copyright (C) 2012-2015 Dan Allen, Ryan Waldron and the Asciidoctor Project
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
-------------------------------------------------------------------------
-
-JQuery License
-
-The following file is used within documentation:
+ANTLR 2 License
 
-    src/spec/assets/css/jquery-2.1.1.min.js
+Antlr2 is released in the public domain.
+See licenses/antlr2-license.txt for details.
```

Comment:
Could you add also the contents of the license file?

Reply:
Shall I add an additional one or refer to the existing ANTLR-2 in nbbuild/licenses?

Reply:
I suppose it's OK. @matthiasblaesing What do you think?